lib/curses.neon

```diff
@@ -12,8 +12,152 @@ DECLARE NATIVE CONST OK: Number
 
 TYPE Window := POINTER
 
+%|----------------------------------------------------------------------
+ |
+ |  Video Attributes
+ |
+ |%
+
+DECLARE NATIVE CONST A_CHARTEXT         : Number
+DECLARE NATIVE CONST A_NORMAL           : Number
+DECLARE NATIVE CONST A_ALTCHARSET       : Number
+DECLARE NATIVE CONST A_RIGHTLINE        : Number
+DECLARE NATIVE CONST A_LEFTLINE         : Number
+DECLARE NATIVE CONST A_INVIS            : Number
+DECLARE NATIVE CONST A_UNDERLINE        : Number
+DECLARE NATIVE CONST A_REVERSE          : Number
+DECLARE NATIVE CONST A_BLINK            : Number
+DECLARE NATIVE CONST A_BOLD             : Number
+DECLARE NATIVE CONST A_ATTRIBUTES       : Number
+DECLARE NATIVE CONST A_COLOR            : Number
+DECLARE NATIVE CONST A_ITALIC           : Number
+DECLARE NATIVE CONST A_PROTECT          : Number
+
+%|----------------------------------------------------------------------
+ |
+ |  Color Macros
```

Comment:
You're commenting the Neon code. Neon doesn't have "macros". You are defining constants. The underlying implementation is not relevant at this level.